e2e/smartvol_ops_test.go

```diff
@@ -316,6 +317,41 @@ func testSmartVolumeDistributeDisperse(t *testing.T) {
 	checkZeroLvs(r)
 }
 
+func editDevice(t *testing.T) {
+	r := require.New(t)
+	peerList, err := client.Peers()
+	r.Nil(err)
+
+	var deviceList []deviceapi.Info
+	var peerID string
+	for _, peer := range peerList {
+		deviceList, err = client.DeviceList(peer.ID.String())
+		if len(deviceList) > 0 {
+			peerID = peer.ID.String()
+			break
+		}
+	}
+
+	if len(deviceList) > 0 {
+		device := deviceList[0]
+		if device.State == "enabled" {
+			err = client.DeviceEdit(peerID, device.Name, "disabled")
+			r.Nil(err)
+		} else if device.State == "disabled" {
+			err = client.DeviceEdit(peerID, device.Name, "enabled")
```

Comment:
I would like to see below-listed tests in this PR
* Disable device and check devices state is disabled
* when all devices are disabled, the volume creation should fail
* Enable all the devices, and check devices is enabled
* when all the devices are enabled, the volume creation should pass.

Reply:
@aravindavk ^^